runtime/near-vm-logic/Cargo.toml

```diff
@@ -58,4 +58,4 @@ name = "test_iterators"
 path = "tests/test_iterators.rs"
 
 [target.'cfg(not(target_arch = "wasm32"))'.dependencies]
-sodiumoxide =  "0.2.5"
+sha2 = "0.8"
```

Comment:
Missing empty line

Reply:
empty line after `sha2 = "0.8"`?

Reply:
Yes, the file should end with empty line.